src/services/CapabilitiesManager.ts

```diff
@@ -136,7 +137,7 @@ export async function setRemoteCapabilities(joinRoomResponse: JoinRoomFullRespon
 	patchTokenMap(joinRoomResponse.data.ocs.data)
 
 	// As normal capabilities update, requires a reload to take effect
-	showError(t('spreed', 'Nextcloud Talk Federation was updated, please reload the page'), {
+	showError(t('spreed', 'Nextcloud Talk Federation was updated.') + '\n' + messagePleaseReload, {
```

Comment:
Something for later, but this should say something like:
> "Nextcloud Talk was updated on the federated server."

And we should have a look if we can make sure it does not yield when you open such a conversation the first time

Reply:
Extracted to https://github.com/nextcloud/spreed/issues/14201

Reply:
Another point regarding `\n`: we discussed with Dorra, if toasts width should be limited upstream to ~750px (80 characters per line is recommended by W3C standard)?